src/components/pages/notice/noticeTemplate.scss

```diff
@@ -0,0 +1,30 @@
+@import 'helsinki/colors';
```

Comment:
Are there predefined values available in the project for the constants used here?

Reply:
Good catch, there are - I'll confirm that they look correct and use those instead.